packages/table/src/Table/Table.stories.tsx

```diff
@@ -0,0 +1,602 @@
+      <TableBody>
+        {data.map((row: any) => (
+          <Row>
+            {columns.map((cellKey: string, index: number) => {
+              return <Cell key={`${cellKey}-${index}`}>{row[cellKey]}</Cell>;
+            })}
+            {row.subRows && row.subRows.map((subRow: any) => {
+              let subRowCellKeys = Object.keys(subRow).filter(x => x !== 'renderExpandedContent' && x !== 'subRows')
+              return (
+                <Row>
+                  {subRowCellKeys.map((cellKey: string) => (
+                    <Cell>
+                      {subRow[cellKey]}
+                    </Cell>
+                  ))}
+                </Row>
+              )
+            })}
+          </Row>
+        ))}
+      </TableBody>
+    </Table>
+  );
+};
+
+export const Sortable = () => {
+  return <>TODO</>
+}
+
+export const Selectable = () => {
+  return <>TODO</>
+}
+
+export const ExpandableContent = () => {
+  const data = makeData(false, 100, 3, 2);
+  const columns = Object.keys(data[0]).filter(x => x !== 'renderExpandedContent' && x !== 'subRows')
+  return (
+    <Table>
+      <TableHead>
+        <HeaderRow>
+          {columns.map((columnName: string) => (
+            <HeaderCell key={columnName} columnName={columnName} />
+          ))}
+        </HeaderRow>
+      </TableHead>
+      <TableBody>
+        {data.map((row: any, index: number) => (
+          <Row>
+            {columns.map((cellKey: string, index: number) => {
+              return <Cell key={`${cellKey}-${index}`}>{row[cellKey]}</Cell>;
+            })}
+            {index % 2 === 0 && (
+              <ExpandedContent>
+                <pre>
+                  {JSON.stringify(row, null, 2)}
+                </pre>
+              </ExpandedContent>
+            )}
+          </Row>
+        ))}
+      </TableBody>
+    </Table>
+  );
+}
+
+export const BasicWithVS = () => {
+  const tableContainerRef = React.useRef<HTMLDivElement>(null);
+  const [data, setData] = React.useState(() => makeData(false, 5000));
+
+  const columns = React.useMemo<Array<ColumnDef<Person>>>(
+    () => [
+      {
+        accessorKey: 'id',
+        header: 'ID',
+        size: 60,
+      },
+      {
+        accessorKey: 'firstName',
+        header: 'First Name',
+        cell: info => info.getValue(),
+      },
+      {
+        accessorFn: row => row.lastName,
+        id: 'lastName',
+        cell: info => info.getValue(),
+        header: () => <span>Last Name</span>,
+      },
+      {
+        accessorKey: 'age',
+        header: () => 'Age',
+        size: 50,
+      },
+      {
+        accessorKey: 'visits',
+        header: () => <span>Visits</span>,
+        size: 50,
+      },
+      {
+        accessorKey: 'status',
+        header: 'Status',
+        size: 90,
+      },
+      {
+        accessorKey: 'progress',
+        header: 'Profile Progress',
+        size: 80,
+      },
+    ],
+    [],
+  );
+
+  const table = useLeafygreenTable<Person>({
```

Comment:
can we make this pattern extend to no virtual rows? we can add an extra arg useVirtualRows or something, so that this API can be compatible either way